### PR TITLE
Closes #17 Apply lazyload on the picture element

### LINKED ISSUE #17: Add lazyload for picture element
The lazyload script can apply lazyload on the `picture` element, so in  the plugin we need to detect any of this element on the page and rewrite the markup to fit.

### PR DESCRIPTION
**Pull request by Tabrisrp**

### CODE REVIEW

File: src/Image.php [hunk 1]
```diff
@@ -50,6 +50,48 @@ public function lazyloadImages($html, $buffer)
...
+            }
+
+            $sources = array_unique($sources, SORT_REGULAR);
+
+            foreach ($sources as $source) {
+                if ($this->isExcluded($source['atts'], $excluded)) {
+                    continue;
+                }
+
+                $lazyload_srcset = str_replace('srcset', 'data-srcset', $source[0]);
```
Comment (Screenfeed) [inline src/Image.php]: I would use a regular expression instead:
```php
$lazyload_srcset = preg_replace('/([\s"'])srcset/i', '\1data-srcset', $source[0]);
```
Reason: https://regex101.com/r/X5d83w/1

File: src/Image.php [hunk 2]
```diff
@@ -50,6 +50,48 @@ public function lazyloadImages($html, $buffer)
...
+     */
+    public function lazyloadPictures($html, $buffer)
+    {
+        preg_match_all('#<picture(?:.*)?>(?<sources>.*)</picture>#iUs', $buffer, $pictures, PREG_SET_ORDER);
+
+        if (empty($pictures)) {
+            return $html;
+        }
+
+        $pictures = array_unique($pictures, SORT_REGULAR);
```
Comment (Screenfeed) [inline src/Image.php]: No strict comparison?

Comment (Tabrisrp) [reply src/Image.php]: doesn't work on multidimensional array

Comment (Screenfeed) [reply src/Image.php]: Right.

**Pull request merged**